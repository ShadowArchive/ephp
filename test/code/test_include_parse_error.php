<?php

include(__DIR__ . "/error.php");

