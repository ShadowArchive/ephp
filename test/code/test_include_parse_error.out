
Parse error: syntax error, unexpected 'is' (T_STRING) in {{CWD}}/test/code/error.php on line 3
